src/test/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionControllerTest.java

```diff
@@ -126,6 +130,38 @@ void  patchObjectionNotFoundExceptionTest() throws  Exception {
         assertEquals(HttpStatus.NOT_FOUND, response.getStatusCode());
     }
 
+    @Test
+    void getObjectionTest() throws Exception {
+        Objection objection = new Objection();
+        objection.setId(OBJECTION_ID);
+        objection.setStatus(ObjectionStatus.OPEN);
+        ObjectionResponseDTO objectionDTO = new ObjectionResponseDTO();
+        objectionDTO.setId(OBJECTION_ID);
+        objectionDTO.setStatus(ObjectionStatus.OPEN);
+        when(objectionMapper.objectionEntityToObjectionResponseDTO(objection)).thenReturn(objectionDTO);
+        when(objectionService.getObjection(any(), any())).thenReturn(objection);
+        when(pluggableResponseEntityFactory.createResponse(any(ServiceResult.class))).thenReturn(
+                ResponseEntity.status(HttpStatus.OK).body(ChResponseBody.createNormalBody(objectionDTO)));
+
+        ResponseEntity<ChResponseBody<ObjectionResponseDTO>> response =
+                objectionController.getObjection(COMPANY_NUMBER, OBJECTION_ID, REQUEST_ID);
+        ChResponseBody<ObjectionResponseDTO> responseBody = response.getBody();
+        ObjectionResponseDTO responseDTO = responseBody.getSuccessBody();
+
+        verify(objectionMapper, times(1)).objectionEntityToObjectionResponseDTO(objection);
+
+        assertEquals(objection.getId(), responseDTO.getId());
+    }
+
+    @Test
+    void getObjectionObjectionNotFoundTest() throws Exception {
```

Comment:
now you've added the catch(exception) block, can you add a test to check that part works as well, so have another test for getObjectionUnexpectedExceptionThrown or something